shop/shop/src/main/java/ro/msg/learning/shop/entities/Location.java

```diff
@@ -0,0 +1,43 @@
+package ro.msg.learning.shop.entities;
+
+import lombok.*;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.Set;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Builder
+@Table(name = "location")
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+public class Location {
+    @Id
+    @Column(unique = true)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    @EqualsAndHashCode.Include
+    private String name;
+    @EqualsAndHashCode.Include
+    private String addressCountry;
+    @EqualsAndHashCode.Include
+    private String addressCity;
+    @EqualsAndHashCode.Include
```

Comment:
why including only these fields? if the excluded fields fewer then maybe exclude those ones if needed and make the code cleaner

Reply:
had some problems at the beginning of the training with inserting data in Orders table, thought this was going to solve the problem but I discovered later the source of the problem and forgot about "cleaning" the code. 😄